packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -383,6 +416,14 @@ export class VisualizerNode extends TypedEventEmitter<DataVisualizerEvents> impl
 		);
 	}
 
+	/**
+	 * Begins the process of applying an edit to a Fluid object.
+	 * @param edit - is a Edit object that describes an edit to a Fluid object.
```

Comment:
100% Josh nitpick: some of the phrasing here feels a bit redundant. I think something like the following would be sufficient / a bit more contextually descriptive:
```suggestion
	 * @param edit - The edit to apply to the associated {@link VisualizerNode.sharedObject}.
```